examples/web/nextjs-chat/src/utils/momento-web.ts

```diff
@@ -15,7 +15,7 @@ export enum EventTypes {
 
 export type ChatMessageEvent = {
   event: EventTypes.MESSAGE;
-  username: string;
+  username?: string;
```

Comment:
This entire demo is wired around `ChatMessageEvent` so i made this optional here even though that wasn't the cleanest thing to do. Would have needed more overhaul if we wanted to change the entire wiring.

Reply:
I dont like the way i've done this at all but was the quickest without a overhaul of the entire example (i maybe wrong in that though).. right now this `type` is constructed when a user is sending a message, so at that point it will be undefined; when a user receives the message from the sender, we popualte this through the `tokenId`.

This type `ChatMessageEvent` is used across the app and the [HTML bodies](https://github.com/momentohq/client-sdk-javascript/blob/main/examples/web/nextjs-chat/src/app/pages/chat-room.tsx#L102) too expect this type when displaying the message.. so i wasn't able to remove it from here without changing the HTML part.